cmd/node/node.go

```diff
@@ -970,16 +968,21 @@ func (app *SpacemeshApp) startSyncer(ctx context.Context) {
 func (app *SpacemeshApp) Start(*cobra.Command, []string) {
 	// we use the main app context
 	ctx := cmdp.Ctx
-
 	// Create a contextual logger for local usage (lower-level modules will create their own contextual loggers
 	// using context passed down to them)
 	logger := log.AppLog.WithContext(ctx)
 
+	hostname, err := os.Hostname()
+	if err != nil {
+		logger.With().Error("error reading hostname", log.Err(err))
+	}
 	logger.With().Info("starting spacemesh",
 		log.String("data-dir", app.Config.DataDir()),
-		log.String("post-dir", app.Config.POST.DataDir))
+		log.String("post-dir", app.Config.POST.DataDir),
+		log.String("hostname", hostname),
```

Comment:
What does `hostname` print if there is an error? Assuming it's nil or an empty string, maybe set it to "(unknown)" or something instead to make it explicit?

Reply:
sorry, i meant to panic here and didn't notice that it was just an error log (as on the line below).
i don't think we need to workaround a potential error. on Linux gethostname doesn't require special permissions, on other OS it probably the same.

Reply:
also created a change for other similar cases https://github.com/spacemeshos/go-spacemesh/pull/2488

Reply:
@lrettig i changed Fatal to Panic to preserve original behavior in this file.